src/cmd/template.go

```diff
@@ -96,9 +99,58 @@ var removeTemplateCmd = &cobra.Command{
 	},
 }
 
+var testTemplateCmd = &cobra.Command{
+	Use:   "test",
+	Short: "Tests service templates",
+	Long:  "Tests service templates. This command must be run in the directory of an exosphere template.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		templateDir, err := os.Getwd()
+		if err != nil {
+			panic(err)
+		}
+		templateName := filepath.Base(templateDir)
+		fmt.Printf("We are about to test the template \"%s\"\n\n", templateName)
+		valid, err := template.IsValidTemplateDir(templateDir)
+		if err != nil {
+			panic(err)
+		}
+		if !valid {
+			fmt.Println("Template fails")
+			os.Exit(1)
+		}
+		tempDir, err := ioutil.TempDir("", "template-test")
+		if err != nil {
+			panic(err)
+		}
+		if err := template.CreateEmptyApp(tempDir); err != nil {
+			panic(err)
+		}
+		appDir := path.Join(tempDir, "my-app")
```

Comment:
I agree that it would make it a lot more clear but I am not sure if we want to do more than just entering empty inputs for the boilr prompts (`"my-app"` is our default app name). I can write a function to enter `"template-test-app"` but `my-app` will be created in `tempDir` somewhere depending what `ioutil.TempDir` returns, and will be deleted after the command finishes running. So exosphere developers will never get to see the folder anyway. Let me know what you think.